src/AuthAction.php

```diff
@@ -118,6 +118,11 @@ class AuthAction extends Action
      */
     private $_cancelUrl;
 
+    /**
+     * @var instance of \yii\web\User 
+     * user component, which might be set in different module
+     */
+    public $user;
```

Comment:
I think it should allow setting ID of the component as well as configuration array instead of user instance only i.e. https://github.com/yiisoft/yii2/blob/master/framework/BaseYii.php#L342